streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/security/impl/AuthenticatorImpl.java

```diff
@@ -59,9 +59,11 @@ public User authenticate(String username, String password, String loginType) thr
 
   private User passwordAuthenticate(String username, String password) {
     User user = usersService.findByName(username);
-    if (user == null || user.getLoginType() != LoginType.PASSWORD) {
-      throw new ApiAlertException(
-          String.format("user [%s] does not exist or can not login with PASSWORD", username));
+    if (user == null) {
```

Comment:
This will cause a bug. In LDAP login mode, if the username doesn't exist when first login, a new user needs to be created. so. we can't simply throw an exception.

Reply:
The `passwordAuthenticate` is used to authenticate the password login, where have to throw error when user doesn't exist, while `ldapAuthenticate` is used for ldap login, and we already have that logic to handle the new user add scenario. Thanks.

Reply:
> The `passwordAuthenticate` is used to authenticate the password login, where have to throw error when user doesn't exist, while `ldapAuthenticate` is used for ldap login, and we already have that logic to handle the new user add scenario. Thanks.

Thank you for your clarification. looks good to me